### ISSUE TITLE
Select line on go to definition

### ISSUE THREAD

**Issue by xrnss** [Col-E/Recaf#438]
## Feature_Name
Highlight

Description of the feature
It's currently quite hard to see what line go to sends you too, as the scrolling can sometimes be incorrect and there may be multiple functions of the same name and it is pain to check parameters. I would like to request that the line is selected so it's easier to see please \:)

**Comment by Col-E** [comment]
https://user-images.githubusercontent.com/21371686/147870774-31c53041-75b8-41fd-928c-5de2b69f4679.mp4

Something like this? Its already implemented in the 3X branch, but its only in a pre-release state at the moment.

**Comment by xrnss** [comment]
Exactly! sorry i didn't know this was already in dev but thank you for letting me know

**Comment by Col-E** [comment]
NP, most of this stuff only is mentioned in the discord and on the temp tracking repo. 

Also for awareness, I'm not currently adding anything new to the 2X releases myself. PR's are welcome but I'm spending my time getting 3X to be ready for general usage. Got a ways to go but it'll be worth it in the end.

**Comment by xrnss** [comment]
Good luck and thank you for your response 🙏

**Comment by Col-E** [comment]
In 4X
